tests/pkg-rendition-paginated/OPF/package.opf

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" version="3.0" unique-identifier="pub-id" prefix="rendition: http://www.idpf.org/vocab/rendition/#">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Package Rendering Metadata</dc:coverage>
+    <dc:creator>Matt Garrish</dc:creator>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:date>2021-11-18</dc:date>
+    <dc:description>Tests on handling overflow content with the "rendition:flow" property set to "paginated". The content should be dynamically paginated by the reading system.</dc:description>
```

Comment:
Maybe say "(not scrolled)" or whatever the alternative is.

Reply:
Yep.